integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -663,36 +714,172 @@ public void testStandaloneManagedRestartAlways() {
+        executeLifecycleScript(START_CLUSTER_SCRIPT, CLUSTER_LIFECYCLE, wrongClusterName, "", false),
+        String.format("Failed to run %s", STOP_CLUSTER_SCRIPT));
+    assertTrue(result.contains("cluster cluster-3 is not part of domain"),
+        "The script shouldn't start a cluster that doesn't exist");
+
+    // verify that the script can not stop a non-exist domain
+    String domainName = "mii-start-policy" + "-123";
+    regex = ".*" + domainName + ".*\\s*not found";
+    result = assertDoesNotThrow(() ->
+        executeLifecycleScript(STOP_DOMAIN_SCRIPT, DOMAIN, null, "", false, domainName),
+        String.format("Failed to run %s", STOP_DOMAIN_SCRIPT));
+    assertTrue(verifyExecuteResult(result, regex),"The script shouldn't stop a domain that doesn't exist");
+
+    // verify that the script can not start a non-exist domain
+    result = assertDoesNotThrow(() ->
+        executeLifecycleScript(START_DOMAIN_SCRIPT, DOMAIN, null, "", false, domainName),
+        String.format("Failed to run %s", START_DOMAIN_SCRIPT));
+    assertTrue(verifyExecuteResult(result, regex),"The script shouldn't start a domain that doesn't exist");
+  }
+
+  /**
+   * Negative test to verify that the sample script can not start a server that exceeds the max cluster size
+   * e.g. the max cluster size is 2, the sample script can't start server 3
+   */
+  @Test
+  @DisplayName("Verify that the sample script can not start a server that exceeds the max cluster size")
+  public void testStartMangedServerBeyondMaxClusyterLimit() {
+    String configServerName = "config-cluster-server3";
+    String dynServerName = "managed-server6";
+
+    // verify that the script can not start a server in config cluster that exceeds the max cluster size
+    String regex = ".*" + configServerName + ".*\\s*is not part";
+    String result =  assertDoesNotThrow(() ->
+        executeLifecycleScript(START_SERVER_SCRIPT, SERVER_LIFECYCLE, configServerName, "", false),
+        String.format("Failed to run %s", START_SERVER_SCRIPT));
+    assertTrue(verifyExecuteResult(result, regex),"The script shouldn't stop a server that is beyong the limit");
+
+    // verify that the script can not start a server in dynamic cluster that exceeds the max cluster size
+    regex = ".*outside the range of allowed servers";
+    result =  assertDoesNotThrow(() ->
+        executeLifecycleScript(START_SERVER_SCRIPT, SERVER_LIFECYCLE, dynServerName, "", false),
+        String.format("Failed to run %s", START_SERVER_SCRIPT));
+    assertTrue(verifyExecuteResult(result, regex),"The script shouldn't stop a server that is beyong the limit");
+  }
+
+  /**
+   * Verify that after the admin server is stopped, the sample script can start or stop a server.
+   */
+  @Disabled("Due to the bug OWLS-86251")
+  @Test
+  @DisplayName("In the absence of Administration Server, sample script can start/stop a managed server")
+  public void testServerRestartMSWithoutAdmin() {
```

Comment:
testServerRestartMSWithoutAdmin --> testServerRestartManagedServerWithoutAdmin

Reply:
Add a test to start/stop a manged server form a dynamic cluster.

Reply:
done